src/helpers/donors.js

```diff
@@ -0,0 +1,10 @@
+
+export default {
+  'steemit': 'gold',
```

Comment:
follow eslint rule. remove single quotes when not needed. this should be as steemit:gold